### PR TITLE
US 1583733 Add missing language IDs - 53

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "console" for sample program output displayed on the console.
- Used "xml" for sample program output that is mostly XML with some text.

Contributes to #2192

### CODE REVIEW

File: docs/visual-basic/programming-guide/concepts/linq/how-to-filter-on-an-attribute-xpath-linq-to-xml.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-filter-on-an-attribute-xpath-linq-to-xml.md]:
```suggestion
```console
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-a-child-element-xpath-linq-to-xml.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-a-child-element-xpath-linq-to-xml.md]:
```suggestion
```console
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-a-list-of-child-elements-xpath-linq-to-xml.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-a-list-of-child-elements-xpath-linq-to-xml.md]:
```suggestion
```console
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-a-union-of-two-location-paths-xpath.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-a-union-of-two-location-paths-xpath.md]:
```suggestion
```console
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-descendant-elements-xpath-linq-to-xml.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-descendant-elements-xpath-linq-to-xml.md]:
```suggestion
```console
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-elements-in-a-namespace.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-elements-in-a-namespace.md]:
```suggestion
```console
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-elements-with-a-specific-attribute.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-elements-with-a-specific-attribute.md]:
```suggestion
```console
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-preceding-siblings-xpath-linq-to-xml.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-preceding-siblings-xpath-linq-to-xml.md]:
```suggestion
```console
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-related-elements-xpath-linq-to-xml.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-related-elements-xpath-linq-to-xml.md]:
```suggestion
```console
```

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-sibling-nodes-xpath-linq-to-xml.md
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-sibling-nodes-xpath-linq-to-xml.md]:
```suggestion
```console
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @v-thpra. A few suggestions.

**Comment by v-thpra** [comment]
@mairaw Thanks for the review suggestions! I implemented them all.

**Review by rpetrusha — DISMISSED** [review]
This looks good, @v-thpra. I'll merge now.

**Comment by rpetrusha** [comment]
I think that this is ready to merge once you approve, @maira.

**Review by mairaw — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**
